CPS.md

```diff
@@ -559,7 +555,7 @@ Investigation into a revocation request will begin within 24 hours of receiving
 
 ### 4.9.6 Revocation checking requirement for relying parties
 
-Relying Parties who cannot or choose not to check revocation status, but decide to rely on a certificate anyway, do so at their own risk.
+Relying parties must verify the validity of certificates via CRL or OCSP prior to relying on certificates. Relying Parties who cannot or choose not to check certificate expiration or revocation status, but decide to rely on a certificate anyway, do so at their own risk.
```

Comment:
I don't think we should say "must verify" here as it sounds like a hard requirement when it isn't (so far as I know). Subscribers are allowed to ignore revocation info if they choose to, they just do it at their own risk. Maybe change "must" to "should"?

Also, we need to capitalize the second word of this paragraph as "Relying Parties" is a defined term.

Reply:
We have a similar "must" statement in 4.5.2:
> Relying Parties must fully evaluate the context in which they are relying on certificates and the information contained in them, and decide to what extent the risk of reliance is acceptable. If the risk of relying on a certificate is determined to be unacceptable, then Relying Parties should not use the certificate or should obtain additional assurances before using the certificate.

Should we update that one as well, or continue to use "must" in both?

Reply:
Good catch, let's change that to "should" as well. I don't see anything in the BRs or the Mozilla CA Policy that requires us to say "must" in this context.

Reply:
Done!